certified-connectors/bitskout/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
+    "capabilities": [],
+    "publisher": "Alexey Zelenkin"
```

Comment:
@saferoombiz please add the stackOwner property.
"stackOwner": "Name of company or organization that connector is connecting to."